docs/docs/telemetry.mdx

```diff
@@ -0,0 +1,122 @@
+---
+id: telemetry
+sidebar_label: Rasa Telemetry
+title: Rasa Telemetry
+---
+
+Rasa uses telemetry to report anonymous usage information. This information is
+essential help improve Rasa Open Source for all users.
+For the team working on Rasa Open Source it is important to understand how
+the product is used, it allows us to properly prioritize our research efforts
```

Comment:
Just a grammatical thing: comma splice. Should be 2 sentences:

For the team working on Rasa Open Source it is important to understand how
the product is used. It allows us to properly prioritize our research efforts